spec/acceptance/ad_units_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource "AdUnits" do
+	explanation "A single advertisement that may be served."
+
+  let!(:ad_unit) { create(:ad_unit) }
```

Comment:
Explain to the #engineering Slack channel why and when to use `!` in test declarations